translations/TranslationServer.js

```diff
@@ -323,6 +323,13 @@ var postHandler = function(data) {
         throw new Error('Unsupported translation schema ' + data.translation);
     }
     var translation = data.transMap[data.transDir][data.translation];
+    var haveDropDefaults = haveDropDefaults = data.dropDefaults && (data.dropDefaults === "true" || data.dropDefaults === "false");
```

Comment:
`var haveDropDefaults = haveDropDefaults =`
Is this duplication a typo?

Reply:
ah i think that's definitely a typo